wagtail/contrib/search_promotions/migrations/0006_searchpromotion_external_link_title_and_more.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.4 on 2023-09-03 09:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailcore", "0089_log_entry_data_json_null_to_object"),
+        ("wagtailsearchpromotions", "0005_switch_query_model"),
```

Comment:
```suggestion
        ("wagtailsearchpromotions", "0006_reset_query_sequence"),
```

This migration can now be used after 006.